### PR TITLE
Error message updates for eCQM

### PR DESCRIPTION
**Pull request by saquino0827**
### Information
- QPPCT-760

### Changes proposed in this PR:
- Update **eCQM section** to **measure section**
- Remove **eCQM** in places where it is not needed.

### Checklist 
- [x] All JUnit tests pass (`mvn clean verify`).
- [n/a] New unit tests written to cover new functionality.
- [n/a] Added and updated JavaDocs for non-test classes and methods.
- [n/a] No local design debt. Do you feel that something is "ugly" after your changes?
- [x] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java [hunk 1]
```diff
@@ -56,13 +56,13 @@
...
 	CLINICAL_DOCUMENT_MISSING_ACI_OR_IA_OR_ECQM_CHILD(23, "Clinical Document Node must have at least one "
-			+ "Aci or IA or eCQM Section Node as a child"),
+			+ "Aci or IA or Measure section Node as a child"),
 	CLINICAL_DOCUMENT_MISSING_PROGRAM_NAME(24, "Clinical Document must have one and only one program name"),
 	CLINICAL_DOCUMENT_INCORRECT_PROGRAM_NAME(25, "The Clinical Document program name `(program name)` is not recognized. Valid "
 		+ "program names are `(list of valid program names)`.", true),
 	CLINICAL_DOCUMENT_CONTAINS_DUPLICATE_ACI_SECTIONS(26, "Clinical Document contains duplicate ACI sections"),
 	CLINICAL_DOCUMENT_CONTAINS_DUPLICATE_IA_SECTIONS(27, "Clinical Document contains duplicate IA sections"),
-	CLINICAL_DOCUMENT_CONTAINS_DUPLICATE_ECQM_SECTIONS(28, "Clinical Document contains duplicate eCQN "
+	CLINICAL_DOCUMENT_CONTAINS_DUPLICATE_ECQM_SECTIONS(28, "Clinical Document contains duplicate measure "
```
Comment (halprin) [inline commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java]: I would capitalize `m` in `measure` to show that this not just any measure section, but _the_ Measure section.

File: commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java [hunk 2]
```diff
@@ -77,7 +77,7 @@
 			+ "Identifier should be specified"),
 	CPC_CLINICAL_DOCUMENT_EMPTY_APM(62, "The Alternative Payment Model (APM) Entity Identifier must not be empty"),
 	CPC_CLINICAL_DOCUMENT_INVALID_APM(63, "The Alternative Payment Model (APM) Entity Identifier is not valid"),
-	CPC_CLINICAL_DOCUMENT_ONE_MEASURE_SECTION_REQUIRED(36, "Must contain one Measure (eCQM) section"),
+	CPC_CLINICAL_DOCUMENT_ONE_MEASURE_SECTION_REQUIRED(36, "Must contain one measure section"),
```
Comment (halprin) [inline commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java]: Same capitalization comment as above.

File: commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java [hunk 3]
```diff
@@ -121,10 +121,10 @@
 	CPC_PLUS_TOO_FEW_QUALITY_MEASURES(65, "CPC+ Submissions must have at least `(Overall CPC+ measure minimum)` of "
 		+ "the following measures: `(Listing of all CPC+ measure ids)`.", true),
 	CPC_PLUS_MISSING_SUPPLEMENTAL_CODE(66, "Missing the `(Supplemental Type)` - `(Type Qualification)` supplemental data for code "
-		+ "`(Supplemental Data Code)` for eCQM measure `(Measure Id)`'s Sub-population `(Sub Population)`", true),
+		+ "`(Supplemental Data Code)` for the measure id `(Measure Id)`'s Sub-population `(Sub Population)`", true),
 	CPC_PLUS_SUPPLEMENTAL_DATA_MISSING_COUNT(67, "Must have one count for Supplemental Data `(Supplemental Data Code)` "
-		+ "on Sub-population `(Sub Population)` for eCQM measure `(Measure Id)`", true),
-	CPC_PLUS_SUBMISSION_ENDED(68, "Your CPC+ submission was made after the CPC+ eCQM submission deadline of "
+		+ "on Sub-population `(Sub Population)` for the measure id `(Measure Id)`", true),
+	CPC_PLUS_SUBMISSION_ENDED(68, "Your CPC+ submission was made after the CPC+ measure section submission deadline of "
```
Comment (halprin) [inline commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java]: Same capitalization comment as above.

File: commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java (hunk 1 above)
Comment (saquino0827) [reply commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java]: 👍

File: commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java [hunk 4]
```diff
@@ -121,10 +121,10 @@
 	CPC_PLUS_TOO_FEW_QUALITY_MEASURES(65, "CPC+ Submissions must have at least `(Overall CPC+ measure minimum)` of "
 		+ "the following measures: `(Listing of all CPC+ measure ids)`.", true),
 	CPC_PLUS_MISSING_SUPPLEMENTAL_CODE(66, "Missing the `(Supplemental Type)` - `(Type Qualification)` supplemental data for code "
-		+ "`(Supplemental Data Code)` for eCQM measure `(Measure Id)`'s Sub-population `(Sub Population)`", true),
+		+ "`(Supplemental Data Code)` for the measure id `(Measure Id)`'s Sub-population `(Sub Population)`", true),
 	CPC_PLUS_SUPPLEMENTAL_DATA_MISSING_COUNT(67, "Must have one count for Supplemental Data `(Supplemental Data Code)` "
-		+ "on Sub-population `(Sub Population)` for eCQM measure `(Measure Id)`", true),
-	CPC_PLUS_SUBMISSION_ENDED(68, "Your CPC+ submission was made after the CPC+ eCQM submission deadline of "
+		+ "on Sub-population `(Sub Population)` for the Measure id `(Measure Id)`", true),
```
Comment (halprin) [inline commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java]: I hate to be that guy, but this `Measure` should be lowercase just like the one above it.

**Pull request merged**